File: app/src/main/java/com/callmangement/EHR/imagepicker/ui/camera/CameraView.java
package com.callmangement.EHR.imagepicker.ui.camera;



import com.callmangement.EHR.imagepicker.model.Image;
import com.callmangement.EHR.imagepicker.ui.common.MvpView;

import java.util.List;

/**
 * Created by hoanglam on 8/22/17.
 */

public interface CameraView extends MvpView {

    void finishPickImages(List<Image> images);
}

File: app/src/main/java/com/callmangement/ehr/imagepicker/ui/camera/CameraView.kt
package com.callmangement.ehr.imagepicker.ui.camera

import com.callmangement.ehr.imagepicker.model.Image
import com.callmangement.ehr.imagepicker.ui.common.MvpView


/**
 * Created by hoanglam on 8/22/17.
 */
interface CameraView : MvpView {
    fun finishPickImages(images: List<Image?>?)
}
